Makefile

```diff
@@ -151,6 +151,16 @@ test-ui-e2e:
 	./hack/install.sh
 	./test/ui-e2e-tests.sh
 
+# Run only kitchensink e2e tests
+test-kitchensink-e2e-testonly:
+	./test/kitchensink-e2e-tests.sh
+
+test-kitchensink-e2e:
+	UNINSTALL_STRIMZI="false" ./hack/strimzi.sh
+	INSTALL_KAFKA="true" ./hack/install.sh
+	./test/kitchensink-e2e-tests.sh
+	./hack/teardown.sh
```

Comment:
We don't need the teardown here. IMO. We already test that [here](https://github.com/openshift-knative/serverless-operator/blob/main/Makefile#L96)
But otherwise this PR looks good!

Reply:
I thought the non-testonly targets do run teardown to cleanup after themselves, not just to test that teardown works.. (but I see that not all of them do,.. so I don't really care either way.)

Reply:
Running teardown in CI is not required. And I think it is there to just test that the teardown works as expected.